### DISCUSSION TITLE
Oobabooga won't start, KeyError

### DISCUSSION THREAD

**Discussion by BooBoo992001** [oobabooga/text-generation-webui#1849] [Q&A]
Ran the Windows installer (alpaca model) and everything seemed fine, but then at the end I get this:

> Traceback (most recent call last):
>   File "C:\AI\oobabooga_windows\text-generation-webui\server.py", line 885, in <module>
>     create_interface()
>   File "C:\AI\oobabooga_windows\text-generation-webui\server.py", line 472, in create_interface
>     with gr.Blocks(css=ui.css if not shared.is_chat() else ui.css + ui.chat_css, analytics_enabled=False, title=title, theme=ui.theme) as shared.gradio['interface']:
>   File "C:\AI\oobabooga_windows\installer_files\env\lib\site-packages\gradio\blocks.py", line 1285, in __exit__
>     self.config = self.get_config_file()
>   File "C:\AI\oobabooga_windows\installer_files\env\lib\site-packages\gradio\blocks.py", line 1261, in get_config_file
>     "input": list(block.input_api_info()),  # type: ignore
>   File "C:\AI\oobabooga_windows\installer_files\env\lib\site-packages\gradio_client\serializing.py", line 40, in input_api_info
>     return (api_info["serialized_input"][0], api_info["serialized_input"][1])
> KeyError: 'serialized_input'

Can anyone tell me what this means and (hopefully) how to fix it?

System:
AMD Ryzen 9 3900XT
NVIDIA RTX 3080 Ti
64GB RAM
Windows 10

**Comment by vicitooo** [comment]
Yes, I have the same thing. I tried with numerous different ways. All the installations ways all lead to this.

**Reply by BooBoo992001** [reply]
See my response to RedLeader below.

**Comment by vicitooo** [comment]
pip install --upgrade gradio

**Reply by BooBoo992001** [reply]
Thank you for replying, but that didn't work.

**Comment by RedLeader721** [comment]
Same problem.  This is happening on new Windows one-click installs.  New install, pip install --upgrade gradio does not fix it, update_windows.bat does not fix it.

**Reply by BooBoo992001** [reply]
Found a Reddit thread on this and it fixed it for me, although at least a couple of others in that thread said it didn't work for them, so YMMV, but:

1. Go to folder where oobabooga_windows is installed and double-click on the cmd_windows.bat file. this should open a command prompt window.

2. Run this command in the command prompt: " pip install gradio==3.28.3 "

3. You can close that command prompt after it finishes and then try restarting again by clicking the start_windows.bat file

**Reply by RedLeader721** [reply]
No change: KeyError: 'serialized_input'

**Comment by BusTopGamer** [comment]
Mine had same issue. solved with pre_layer setting under Model Tab. increase it to something suggested between 20-30 if GPU has 8gb VRAM.
